### DISCUSSION TITLE
[flip] Name overflowing directions

### DISCUSSION THREAD

**Discussion by xgebi** [floating-ui/floating-ui#2756] [General]
When I get an object from middleware under `flip` I get:

```
{ 
  index: 1,
  overflows: [{
    overflows: [10, -274.09375286102295, -268.90624713897705]
    placement: "left"
  }]
}
```

1. use `flip` middleware
2. add console log on middlewareData
3. add tooltip that overflows

**Expected behavior**

It would be easier to use if I knew in which direction the tooltip overflowed. As in

```
{ 
  index: 1,
  overflows: [{
    overflows: {
      top: 10, 
      bottom: -274.09375286102295, 
      right: -268.90624713897705
    }
    placement: "left"
  }]
```

**Context:**

- I'm using floating-ui 1.5.4

**Comment by atomiks** [comment]
What's the use case for this? (Converted to Feature Request discussion)
